docs/src/style.md

```diff
@@ -179,6 +179,19 @@ some_local_variable = ...
 some_file.jl # Except for ModuleName.jl.
 ```
 
+Begin private functions and constants with an underscore. All other names should
+be exported.
```

Comment:
Hmm. Since it's not exported, it should begin with an underscore. But since it acts like a local variable, it should never begin with an underscore...

```julia
function _private_jump_function(x)
    function _internal_function(y)
        return x + y
    end
    return _internal_function(2 * x)
end

function _private_jump_function(x)
    function internal_function(y)
        return x + y
    end
    return internal_function(2 * x)
end
```

Looking at that, the underscore seems better to me...

Reply:
To start out we can say that this point applies only to objects in the scope of a module.